e2e/README.md

```diff
@@ -60,15 +60,28 @@ following environment variables can be exported to customize kubernetes deployme
 - creating kubernetes  cluster
 
 ```console
-$./minikube.sh up
+From the ceph-csi root directory, run:
+
+$ ./scripts/minikube.sh up
 ```
 
 - Teardown kubernetes cluster
 
 ```console
-$./minikube.sh clean
+$ ./scripts/minikube.sh clean
```

Comment:
Other commands use `[$]`, probably because the markdown linter detects this as an issue:
```
./e2e/README.md:84: MD012 Multiple consecutive blank lines
./e2e/README.md:70: MD014 Dollar signs used before commands without showing output
./e2e/README.md:80: MD014 Dollar signs used before commands without showing output
./e2e/README.md:130: MD014 Dollar signs used before commands without showing output
./e2e/README.md:156: MD014 Dollar signs used before commands without showing output
./e2e/README.md:162: MD014 Dollar signs used before commands without showing output
```

Please run `make containerized-test` locally before pushing the next updated version of this PR.

Reply:
thanks for the suggestion. I Will check on this.

Reply:
May be `./scripts/lint-extras.sh lint-markdown ` should be good.

Reply:
you can pass `TARGET=lint-markdown` on the `make containerized-test` commandline too